src/ServerConfigParams.ts

```diff
@@ -78,7 +79,7 @@ const serverDefaultConfiguration: ServerConfigParams = {
     url: 'http://localhost:8090',
     port: 0,
     workdir: '',
-    checkInterval: 10000,
+    checkInterval: 1000,
```

Comment:
Why do we want to reduce the default check interval? 1 seconds may be too much

Reply:
my bad, was just for testing purposes, let me change it.

Reply:
updated